docs/14-i18n.md

```diff
@@ -0,0 +1,34 @@
+# I18n
+
+The app has two languages:
+* **UI language**
+* **Content language**
+
+## UI language
+
+The UI language is always the primary system language and cannot be changed.
+
+All UI and navigation elements are using the UI language and the corresponding layout direction, i.e. LTR or RTL.
```

Comment:
We should decide here for a naming of the elements. I think UI elements is fine.

Reply:
Looking good to me.